path-manager/app/services/PathStore.scala

```diff
@@ -133,6 +199,18 @@ object PathStore extends Logging {
     }
   }
 
+  def deleteAliasesAssociatedWithPath(path: String) = {
+    logger.debug(s"deleting alias records for path [$path]")
+    val maybeItem = Option(Dynamo.pathsTable.getItem("path", path)).map(PathRecord(_))
+    maybeItem match {
+      case Some(PathRecord(dynamoPath, id, "alias", _ )) if dynamoPath == path => {
+        logger.debug(s"deleting alias path [$path] for id [${id}]")
+        Dynamo.pathsTable.deleteItem("path", path)
+        logger.debug(s"deleted path records for [$path] successfully")
```

Comment:
if we are keeping this then singular is probably more accurate...
```suggestion
        logger.debug(s"successfully deleted alias path [$path] for id [${id}]")
```